manage.py

```diff
@@ -148,6 +148,57 @@ def cf_startup():
         subprocess.Popen(["python", "cli.py", "refresh_materialized"])
 
 
+def check_long_queries(minutes):
+    """
+    Check for queries running longer than interval, default is 5
+    """
+
+    SQL = """
+        SELECT *
+        FROM pg_stat_activity
+        WHERE datname <>'rdsadmin'
+        and usename ='fec_api'
+        and lower(query) like 'select %'
+        and lower(query) not like '%refresh%'
+        and lower(query) not like '%rollback%'
+        and (now() - pg_stat_activity.query_start) >= interval :minutes
+        order by pg_stat_activity.query_start desc;
+        """
+
+    results = db.engine.execute(sa.text(SQL), minutes=f"{minutes} minutes")
+    rows = results.fetchall()
+
+    for row in rows:
+        print(row)
+    total_rows = results.rowcount
+    print("Total: ", total_rows)
+
+
+def clear_long_queries(minutes):
+    """
+    Check for queries running longer than interval minutes, default is 5
+    """
+
+    SQL = """
+        SELECT pg_terminate_backend(pid)
+        FROM pg_stat_activity
+        WHERE datname <>'rdsadmin'
+        and usename ='fec_api'
+        and lower(query) like 'select %'
+        and lower(query) not like '%refresh%'
+        and lower(query) not like '%rollback%'
+        and (now() - pg_stat_activity.query_start) >= interval :minutes
+        order by pg_stat_activity.query_start desc;
+        """
+
+    results = db.engine.execute(sa.text(SQL), minutes=f"{minutes} minutes")
+    rows = (results.fetchall())
+    total_rows = results.rowcount
+
+    print(rows)
+    print("Terminated ", total_rows, " queries")
```

Comment:
Consider posting slack message in alerts# or bots# when these two task are executed. Those messages come in handy when triaging issues.

Reply:
During the creation of the ticket, it was decided that checking the logs would be the best way to get the queries both because they are long and because the bots channel is busy. As a compromise, I added just the termination info to the bots channel. Open to changing if the team feels strongly.